1.0.0

```diff
@@ -0,0 +1,8 @@
++ react-native-gesture-handler@2.6.1
```

Comment:
what is the purpose of this file

Reply:
this is co-dependency with react-navigation/stack but seem not to use it because we use react-navigation/native-stack instead

Reply:
Even use react-navigation/stack or any other react-navigation libs. This file and the file which named 2.8.0 are useless in project